projects/igniteui-angular/src/lib/tabs/tab-header.directive.ts

```diff
@@ -7,6 +7,10 @@ import { IgxTabHeaderBase, IgxTabsBase } from './tabs.base';
 @Directive()
 export abstract class IgxTabHeaderDirective implements IgxTabHeaderBase {
 
+    /** @hidden */
+    @HostBinding('attr.role')
+    public role = 'tabitem';

```

Comment:
Role should be `tab`.
```suggestion
    public role = 'tab';

```

Reply:
Done